pkg/test/framework/components/istio/operator.go

```diff
@@ -684,3 +732,76 @@ func deployCACerts(workDir string, env *kube.Environment, cfg Config) error {
 	}
 	return nil
 }
+
+func createIstiodEndpoint(disvoveryAddress string, cfg Config, cluster resource.Cluster) error {
```

Comment:
This looks a lot like what we do for `istiod-remote` except for the webhook bit.

Reply:
This is to make sure we get correct discoveryAddress and CAAddr for workloads in istiodRemote.